src/applications/vaos/new-appointment/sass/_calendar.scss

```diff
@@ -1,8 +1,14 @@
-.vaos-calendar__loading {
+.vaos-calendar__disabled {
   opacity: 0.3;
+  pointer-events: none;
+  background: none !important;
+  -webkit-user-select: none;  /* Chrome all / Safari all */
+  -moz-user-select: none;     /* Firefox all */
+  -ms-user-select: none;      /* IE 10+ */
+  user-select: none;          /* Likely future */
```

Comment:
Disables click events for elements behind the disabled overlay

Reply:
Does it disable keyboard nav? I noticed that the other day with the current overlay, you can tab through the calendar still.

We also may want to consider how screen reader users are being told the calendar is disabled, though that's sort of similar to the keybaord nav question.